ui/packages/ace/package.json

```diff
@@ -63,6 +63,7 @@
     "worker-loader": "1.1.1"
   },
   "devDependencies": {
+    "cli-color": "1.2.0",
```

Comment:
How would you feel about using [chalk](https://www.npmjs.com/package/chalk)? It's already part of our dependency graph, so we wouldn't be bringing in new dependencies.